src/fsverity/ioctl.rs

```diff
@@ -17,6 +17,15 @@ pub enum MeasureVerityError {
     InvalidDigestSize { expected: u16 },
 }
 
+/// Enabling fsverity failed.
+#[derive(Error, Debug, PartialEq)]
+pub enum EnableVerifyError {
+    #[error("Filesystem not supported by fs-verity")]
```

Comment:
I'd describe the situation with slightly different language: the filesystem doesn't support fs-verity.  The enablement code is in the filesystem, not in fs-verity.